docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md

```diff
@@ -56,135 +56,112 @@ This tutorial assumes that:
         Class.forName("org.postgresql.Driver");
-        try (Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte", "yugabyte", "yugabyte");
-            Statement stmt = conn.createStatement();) {
+        Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte",
+                                                      "yugabyte", "yugabyte");
+        Statement stmt = conn.createStatement();
+        try {
             System.out.println("Connected to the PostgreSQL server successfully.");
-
-            String createTableQuery = "CREATE TABLE IF NOT EXISTS employee(id int primary key, name varchar, age int, language text) ";
-            stmt.executeUpdate(createTableQuery);
+            stmt.execute("DROP TABLE IF EXISTS employee");
+            stmt.execute("CREATE TABLE IF NOT EXISTS employee" +
+                        "  (id int primary key, name varchar, age int, language text)");
             System.out.println("Created table employee");
 
-            String insertQuery = "INSERT INTO employee (id, name, age, language) VALUES (1, 'John', 35, 'Java');";
-            stmt.executeUpdate(insertQuery);
-            System.out.println("Inserted data: INSERT INTO employee (id, name, age, language) VALUES (1, 'John', 35, 'Java');");
+            String insertStr = "INSERT INTO employee VALUES (1, 'John', 35, 'Java')";
+            stmt.execute(insertStr);
+            System.out.println("EXEC: " + insertStr);
 
             ResultSet rs = stmt.executeQuery("select * from employee");
-            while (rs.next())
-                System.out.println("Query returned: "+ "name=" + rs.getString(2) + ", age=" + rs.getString(3) + ", language=" + rs.getString(4));
+            while (rs.next()) {
+              System.out.println(String.format("Query returned: name = %s, age = %s, language = %s",
+                                              rs.getString(2), rs.getString(3), rs.getString(4)));
+            }
         } catch (SQLException e) {
-            System.err.println(e.getMessage());
-
+          System.err.println(e.getMessage());
         }
+      }
     }
-}
-
-```
-
-### Build the project
-
-To build the project, run the following `mvn package` command.
-
-```sh
-$ mvn package
-```
-
-You should see a `BUILD SUCCESS` message.
-
-### Run the application
+    ```
 
-To run the application , run the following command.
+1. Run your new program.
 
-```sh
-$ java -cp "target/hello-world-1.0.jar:target/lib/*" com.yugabyte.sample.apps.YBSqlHelloWorld
-```
+    ```sh
+    $ mvn install exec:java -Dexec.mainClass=com.yugabyte.HelloSqlApp
+    ```
 
-You should see the following as the output.
+    You should see the following as the output:
 
-```
-Created table employee
-Inserted data: INSERT INTO employee (id, name, age, language) VALUES (1, 'John', 35, 'Java');
-Query returned: name=John, age=35, language: Java
-```
+    ```output
```

Comment:
Missing the first line in the output.  The posted code should also output the following text.
```
Connected to the PostgreSQL server successfully.
```

Reply:
added back in :)